src/db/ebb/globalShortcutManager.ts

```diff
@@ -60,43 +60,57 @@ export const updateGlobalShortcut = async (newShortcut: string): Promise<void> =
     logError('(Global) Cannot update shortcut before initialization')
     return
   }
+
+  const currentShortcut = await getCurrentShortcutFromDb()
   if (newShortcut === currentShortcut) {
     return
   }
 
   try {
-    await unregisterShortcutTauri(currentShortcut)
+    if (currentShortcut) {
+      await unregisterShortcutTauri(currentShortcut)
+    }
+    
+    if (newShortcut) {
```

Comment:
Almost there! I'd do this instead to check if newShortcut has been passed/has a value:

```
  if (newShortcut) return;
  const currentShortcut = await getCurrentShortcutFromDb()
  if (newShortcut === currentShortcut) {
    return
  }

  try {
    if (currentShortcut) {
      await unregisterShortcutTauri(currentShortcut)
    }

    await registerShortcutTauri(newShortcut, (event) => {
      if (event.state === 'Pressed') {
        emit(SHORTCUT_EVENT)
      }
    })

    await saveShortcut(newShortcut)
    
    ```